users/serializers_test.py

```diff
@@ -173,3 +178,16 @@ def test_create_user_serializer(
         mock_user_sync.assert_called_with(user.id)
     else:
         mock_user_sync.assert_not_called()
+
+
+def test_update_email_change_request_existing_email(user):
```

Comment:
Can you mock the task and assert that it is called with the correct args?

Reply:
already done that test in `courseware/tasks/tasks_test.py`.